lib/optional_query_cache/version.rb

```diff
@@ -0,0 +1,11 @@
+require 'active_record'
```

Comment:
@mloughran's solution is the way to go if you're set on the current hierarchy, but I think the problem is the fact that your namespace is a subclass. The alternative would be to create a separate top-level module as a namespace for everything, e.g.:

```ruby
module OptionalQueryCache
  class Version
    # ...
  end
end

module OptionalQueryCache
  class QueryCache < ActiveRecord::QueryCache
    # ...
  end
end
```